### DISCUSSION TITLE
古い未更新記事へ編集リクエストをいただいた時の遷移

### DISCUSSION THREAD

**Discussion by simon-ritchie** [increments/qiita-discussions#228] [要望・アイデア・不具合]
見間違いや勘違いだったら申し訳ございません・・・！

長いこと更新していなかった記事に対して編集リクエストをいただいたのですが、

・編集リクエストを受け入れる画面でお礼のコメントを書いて受け入れのボタンを押す
↓
・記事が古かった影響か以前あったマークダウンパーサー変更による記事のプレビュー（編集）画面になる
↓
・問題無さそうだったので記事の保存をクリックする
↓
・記事は保存されるものの編集リクエストの受け入れがされておらず、お礼用に書いたコメントなども反映されていない？

といった感じのものが発生しました！お礼のコメントの書いた分が消えてしまったり、編集リクエストが受け入れになっておらずユーザーが勘違いしたり・・・するかもしれないためこの流れの場合編集リクエストもちゃんと受け入れが反映されると自然でUXが良いかもしれません・・・！

※編集リクエストが無くなってしまって再現ができないのでもし勘違いでしたらスルーいただけますと幸いです！

**Comment by tanaka-asami415** [comment]
@simon-ritchie 

お問い合わせありがとうございます🙇

ひとつ確認させてください🙏
編集リクエスト一覧にも残っていない状態でしょうか？

画像のメニューは、ご自身のアイコン画像をクリックいただくと表示されます。
「編集リクエスト」をクリックいただくと、一覧が表示されます🙇

<img width="330" alt="ScreenShot 2022-05-23 13 04 37" src="https://user-images.githubusercontent.com/103916185/169771930-7812d7e7-e9f8-4c57-8dad-7670618bd8b4.png">

**Reply by simon-ritchie** [reply]
@tanaka-asami415 さん

はい、現在はマージがされていない編集リクエストは残っていない形となります！

・編集リクエストを受け入れる画面でお礼のコメントを書いて受け入れのボタンを押す
↓
・記事が古かった影響か以前あったマークダウンパーサー変更による記事のプレビュー（編集）画面になる
↓
・問題無さそうだったので記事の保存をクリックする
↓
・記事は保存されるものの編集リクエストの受け入れがされておらず、お礼用に書いたコメントなども反映されていない？
↓
・編集リクエスト一覧画面に遷移するとマージがされておらず残っている
↓
・もう一度マージ時のコメントなどを書いて対応する
↓
・今度は（既に一度更新済みのため）記事のプレビュー（編集）画面にはならずに進む
↓
・編集リクエストが反映され、コメントなどにも反映されており、編集リクエスト一覧画面からもマージ待ちのものは消えた

という感じになっております！

![image](https://user-images.githubusercontent.com/25178045/169805875-e10a2613-e18d-4aeb-ae7a-19d93e525840.png)

**Comment by tanaka-asami415** [comment]
@simon-ritchie 

詳細の追記をありがとうございます！状況理解できました🙏

おそらくですが、昔に作られた下書きが残っており、
1回目はその下書きが反映されただけで、リクエストが反映されなかったのではないかと推察します。
下書きが残っていると編集リクエストの採用ができない仕様となっております。

>この流れの場合編集リクエストもちゃんと受け入れが反映されると自然でUXが良いかもしれません・・・！

確かにユーザーさまを混乱させてしまうと思いますので、今後どこかのタイミングで改善したいと思います🙇
しばらくはご不便をおかけしてしまいますが、お待ちいただけると幸いです🙏

引き続き何かありましたら、ご意見お待ちしております🙇

**Reply by simon-ritchie** [reply]
@tanaka-asami415 さん

> 下書きが残っていると編集リクエストの採用ができない仕様となっております。

あ、そういった仕様だったのですね・・・！パーサー関係？かと勘違いしておりました・・・！

> 確かにユーザーさまを混乱させてしまうと思いますので、今後どこかのタイミングで改善したいと思います🙇

ありがとうございます！お手数ですがよろしくお願い申し上げます・・・！🙇
